ruby/vampires.rb

```diff
@@ -0,0 +1,77 @@
+attempts = 0
+
+until attempts == 4
+	attempts += 1
+
+puts "What is your name?"
+name = gets.chomp
+	
+
+def age_finder(x, y)
+		 (2016 - y.to_i) == x.to_i
+end
+
+puts "How old are you? What year were you born?"
+age = gets.chomp
+year = gets.chomp
+
+	
+puts "The cafeteria serves garlic bread. Should we order some for you?"
+garlic_bread = gets.chomp
+
+	if garlic_bread == "yes"
+	  garlic_bread = true
+	else
+		garlic_bread == "no"
+	  garlic_bread = false
+	end
+
+puts "Would you like to enroll in the company's health insurance?"
+insurance = gets.chomp
+
+	if insurance == "yes"
+	  insurance = true
+	elsif insurance == "no"
+	  insurance = false
+	else insurance == nil
+		insurance = false
+	end
+
+
+if age_finder(age.to_i, year.to_i) && (garlic_bread || insurance)
+	puts "Probably not a vampire"
+elsif (age_finder(age.to_i, year.to_i)) &&  (garlic_bread == "no") && (insurance ==nil)
+	puts "Probably a vampire"
+elsif (age_finder(age.to_i, year.to_i)) &&  (garlic_bread == "no" && insurance =="no")
+	puts "Almost certainly a vampire."
+elsif (name ==  "Drake Cula" || name == "Tu Fang") && (age_finder(age.to_i, year.to_i)) &&  (garlic_bread || insurance) 
+	puts "Def a vampire."
```

Comment:
Above I mentioned using a variable as your results and setting it to different strings/outcomes based on the conditionals met. This is cleaner and more reliable than a `puts` statement.